packages/hash/design-system/src/avatar.tsx

```diff
@@ -53,6 +53,8 @@ export const Avatar: FunctionComponent<AvatarProps> = ({
             lineHeight: 1,
           }}
         >
+          {/** @todo what to do about empty title */}
+          {/* eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing */}
           {(title || "User").charAt(0).toUpperCase()}
```

Comment:
Hey Yusuf – I'm purposefully not making changes now where ever possible. This is to make merging this simpler / so it doesn't fall to the same fate as #916. This PR came out of reviewing PRs and not wanting to bikeshed about this issue, but wanting to prevent it nonetheless. 

I think people can fix this as they re-encounter this code.